pkg/oomstore/types/feature.go

```diff
@@ -87,6 +87,31 @@ func (l FeatureList) Find(find func(*Feature) bool) *Feature {
 	return nil
 }
 
+func (l FeatureList) GroupIDs() (ids []int) {
+	for _, f := range l {
+		ids = append(ids, f.GroupID)
+	}
+	return ids
+}
+
+func (l FeatureList) Any(predicate func(*Feature) bool) bool {
+	for _, f := range l {
+		if predicate(f) {
+			return true
+		}
+	}
+	return false
+}
+
+func (l FeatureList) All(predicate func(*Feature) bool) bool {
```

Comment:
do we still need `Any` and `All`?

Reply:
don't use it now, but these are two useful methods 🤔   consider keeping them?

Reply:
let's keep whatever we're using. Adding them back only when needed.

Reply:
done